Libraries/Animated/src/Interpolation.js

```diff
@@ -163,7 +163,23 @@ function interpolate(
   return result;
 }
 
+// Borrowed from http://stackoverflow.com/questions/5623838/rgb-to-hex-and-hex-to-rgb
+function hexToRgba(
+  input: string
+): string {
+  // Expand shorthand form (e.g. "03F") to full form (e.g. "0033FF")
+  var shorthandRegex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i;
```

Comment:
Yeah indeed, wanted to use {3} but then found that out too :stuck_out_tongue_closed_eyes: